server/controllers/user/lib/user.ts

```diff
@@ -107,6 +107,11 @@ export const addUserRole = (userId: UserId, role: UserRole) => db.update(userId,
 
 export const removeUserRole = (userId: UserId, role: UserRole) => db.update(userId, removeUserDocRole(role))
 
+export function userHasAdminRole (user: User) {
```

Comment:
I'm not much in favour of this function, It would be preferable to use `getUserAccessLevels`